.github/workflows/ci.yml

```diff
@@ -297,3 +297,54 @@ jobs:
           hub release create -m $tag_name $tag_name $(for f in cachepot-*.tar.gz*; do echo "-a $f"; done)
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+
+  deploy-gh-pages:
+    runs-on: ubuntu-20.04
+
+    steps:
+    - name: Checkout local repo
+      uses: actions/checkout@v2
+
+    - name: Clone cachepot repo
+      uses: actions/checkout@v2
+      with:
+        repository: paritytech/cachepot
+        path: .tmp_cachepot
+
+    - name: Setup mdbook
+      uses: peaceiris/actions-mdbook@v1
+      with:
+        mdbook-version: 'latest'
+
+    - name: Setup mdbook-linkcheck
+      run: |
+        cargo install mdbook-linkcheck
+
+    - name: Use Rust toolchain
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: stable
+
+    - name: Setup mdbook-graphviz
+      run: |
+        sudo apt install -y graphviz
+        cargo install mdbook-graphviz
+
+    - name: Setup mdbook-mermaid
+      run: |
+        cargo install mdbook-mermaid
```

Comment:
question: FWIU mermaid and graphviz are two ways to visualize graphs - do we need to eventually settle on one for consistency or do we want to use both?

Reply:
We don't know at this point, and both are useful and suffer a slightly different purpose, depending on the level of control that is needed. Right now we are not using either, but I don't want to revisit the mdbook/ci topic anytime soon, but we should formalize the flow graphs soon and I'd like to keep the choices open.